test/tic_tac_toe/core_test.clj

```diff
@@ -11,31 +11,44 @@
             [tic-tac-toe.protocol.input :refer :all]
             [tic-tac-toe.protocol.display :refer :all]))
 
-(deftest game-iterattion-test 
-   (let [ board-winner [["_" "o" "o" ]["x" "_" "_" ]["_" "_" "_" ]]
+; (defn latest-diplayed-state (atom []))
+
+; (defrecord DummyDisplay []
+;   DisplayProtocol
+;   (display-state [display board]
+;     (assoc @latest-diplayed-state board))
+;   (display-winner [display board]))
+
+(deftest game-iterattion-test
```

Comment:
I noticed you have a decent amount of duplication in your tests. Please look into having nested test contexts so you can have parent `let`s that allow you to define variables that are used multiple different tests
http://thornydev.blogspot.com/2012/09/before-and-after-logic-in-clojuretest.html